trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -132,16 +132,22 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+`tracestate` is a Structured header in accordance to [Draft of Structured Headers for HTTP](http://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html). Its value MUST be a dictionary ([Draft of Structured Headers for HTTP](http://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html), Section 4.1).
+
+Maximum length of a combined header MUST be less than 512 bytes. If the maximum length of a combined header is more than 512 bytes it SHOULD be ignored.
```

Comment:
how did this 512 number come up? Why 512? HTTP Cookies are 4kb

Reply:
https://github.com/w3c/distributed-tracing/blob/master/trace_context/HTTP_HEADER_FORMAT_RATIONALE.md#size-limits

Idea was that the size should be small enough so it will fit into messaging systems metadata for most systems. Like queues and such. It was a main concern against using the correlation context

Reply:
I suggest adding the reason to the spec.

Reply:
Added comment into the field definition. Please take a look

Reply:
That rationale is vague and distracting. Can you put a concrete place where 512 would be a limitation? even one or two is better than something that a lot of people find distractingly small.

Reply:
Should the header be ignored or truncated?

Reply:
Truncation can only be formalized with the ordering (issue #80). I intentionally make a language `SHOULD` to allow for advanced implementations.

Also see mention of it in #110. I mention that for privacy concerns you may whitelist keys. But this is highly discouraged